dev/devicelab/bin/tasks/entrypoint_dart_registrant.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Don't I need to register this task somewhere?  I'm not sure what the latest rules are, it used to be in the infra repo but I know they wanted to move it.

Reply:
In ci.yaml. e.g. https://github.com/flutter/flutter/blob/533a5dde96b35798c501811f3ba378b53bc4b7e4/.ci.yaml#L2523-L2544

Reply:
thanks!